resources/linux/rpm/dependencies.json

```diff
@@ -62,7 +62,8 @@
 		"libc.so.6(GLIBC_2.9)(64bit)",
 		"libxcb.so.1()(64bit)",
 		"libxkbfile.so.1()(64bit)",
-		"libsecret-1.so.0()(64bit)"
+		"libsecret-1.so.0()(64bit)",
+		"libgbm.so.1()(64bit)"
 	],
 	"aarch64": [
```

Comment:
What about adding it to the other architectures ?

Reply:
I wouldn't know how to verify that, @joaomoreno

Reply:
I can do that once I get to https://github.com/microsoft/vscode/issues/106473